packager/react-packager/src/Server/index.js

```diff
@@ -483,7 +483,7 @@ class Server {
   }
 
   _processAssetsRequest(req, res) {
-    const urlObj = url.parse(req.url, true);
+    const urlObj = url.parse(decodeURIComponent(req.url), true);
```

Comment:
@ide Yes, it should use decodeURI.
http://stackoverflow.com/questions/747641/what-is-the-difference-between-decodeuricomponent-and-decodeuri